.github/CODEOWNERS

```diff
@@ -12,4 +12,11 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
+# Global owners, will be the owners for everything in the repo.
 * @open-telemetry/operator-approvers
+
+# AutoInstrumentation owners
+# TBD
+
+# Target Allocator owners
+cmd/otel-allocator @open-telemetry/operator-ta-maintainers
```

Comment:
@tigrannajaryan thanks for handling https://github.com/open-telemetry/community/issues/1218.

However this file is marked with errors:

![Screenshot of Use the newly created TA codeowners by jaronoff97 · Pull Request #1145 · open-telemetry_opentelemetry-operator](https://user-images.githubusercontent.com/5618424/194384552-0bf4f7cd-6e75-497d-ae88-5c362bda925a.jpg)

It seems like the group @open-telemetry/operator-ta-maintainers should have write access to this repo.

Reply:
We give Write access to the entire repo only to Approvers. What you are asking for is outside of the current policies so I can't do it. Either find a different way or create an issue in the community repo to change the policies/make exception.